nextjs-app/components/pages/main/Banner.tsx

```diff
@@ -0,0 +1,110 @@
+import Image from "next/image";
+import { twMerge } from "tailwind-merge";
+import RegisterNavigatorButton from "@/components/common/Button/RegisterNavigatorButton";
+import RegisterSailorButton from "@/components/common/Button/RegisterSailorButton";
+import CountdownTimer from "@/components/common/CountdownTimer";
+
+const Banner = () => {
+  return (
+    <div
+      className={twMerge(
+        "relative pt-11",
+        "w-full h-[calc(100vh-80px)] md:h-[calc(100vh-88px)] lg:h-[calc(100vh-94px)]"
+      )}
+    >
+      {/** Text Area */}
+      <div className="flex justify-center lg:justify-start">
+        <div className="lg:ml-[144px] flex flex-col items-center lg:items-start">
+          <div className="text-center lg:text-start text-h1 text-neutral-10 mb-4">
+            <div>在職涯地圖上</div>
+            <div>點亮自己的航線</div>
+          </div>
+          <div>
+            <CountdownTimer />
+          </div>
+          <div className="w-fit flex flex-col justify-center gap-4 mt-7">
+            <RegisterSailorButton />
+            <RegisterNavigatorButton />
+          </div>
+        </div>
+      </div>
+      {/** Animation elements for desktop */}
+      <div
+        className={twMerge(
+          "hidden lg:block",
+          "absolute bottom-8 right-0 -z-30",
+          "w-[calc(70vh/614*1003)] h-[70vh]"
+        )}
+      >
+        <Image
+          src="/images/index-banner/boat.png"
+          alt="boat"
+          fill
+          className={twMerge(
+            "hidden lg:block",
+            "w-full h-full object-contain object-right-bottom",
+            "motion-translate-y-loop-[15px] motion-duration-2000"
```

Comment:
```suggestion
            "motion-translate-y-loop-[8px] motion-duration-2000 motion-ease-in-out"
```

I tried less distance to translate and different timing function, it seems more natural.
But I believe we need to check this effect with designer